Tests/ValetIntegrationTests/ValetIntegrationTests.swift

```diff
@@ -365,6 +365,10 @@ class ValetIntegrationTests: XCTestCase
 
     func test_stringForKey_withDifferingIdentifierInSameAccessGroup_throwsItemNotFound() throws
     {
+        guard testEnvironmentIsSignedOrDoesNotRequireEntitlement() else {
+            return
+        }
```

Comment:
on recent Xcodes, shared group valets require a signed environment. Which makes sense... but is annoying.